OpenRA.Mods.Common/Lint/CheckTranslationReference.cs

```diff
@@ -61,8 +61,12 @@ public void Run(Action<string> emitError, Action<string> emitWarning, ModData mo
 
 					foreach (var entry in result.Entries)
 					{
-						if (!referencedKeys.Contains(entry.GetId()))
-							emitWarning($"Unused key `{entry.GetId()}` in {file}.");
+						var key = entry.GetId();
+						var isReusableTerm = entry.GetType() == typeof(AstTerm);
+
+						// Don't flag definitions referenced (only) within the .ftl definitions as unused
```

Comment:
I meant for this comment to explain the `isReusableTerm` definition, since that otherwise is not obvious (at least to me).